openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/MetadataToESConfigForm.tsx

```diff
@@ -44,62 +60,99 @@ export default function MetadataToESConfigForm({
   };
 
   return (
-    <Form layout="vertical">
-      <Form.Item label="caCerts">
+    <Form className="metadata-to-es-config-form" layout="vertical">
+      <Form.Item label={startCase('caCerts')}>
+        <Text className="input-field-descriptions">
+          {t('message.field-ca-certs-description')}
+        </Text>
         <Input
           value={metadataToESConfig?.caCerts}
           onChange={handleCaCertsChange}
         />
       </Form.Item>
-      <Form.Item label="regionName">
+      <Form.Item label={startCase('regionName')}>
+        <Text className="input-field-descriptions">
+          {t('message.field-region-name-description')}
+        </Text>
         <Input
           value={metadataToESConfig?.regionName}
           onChange={handleRegionNameChange}
         />
       </Form.Item>
```

Comment:
Can we use the name attribute for inputs and instead of using onChange for each input use form `onFinish` prop for getting the data?